src/js/api/config.js

```diff
@@ -104,7 +105,7 @@ const Config = function(options, persisted) {
     let rateControls = config.playbackRateControls;
 
     if (rateControls) {
-        let rates = [0.5, 1, 1.25, 1.5, 2];
+        let rates = config.playbackRates;
```

Comment:
Also, `playbackRates` should be used to reflect the final list of rates moving forward:
```
config.playbackRateControls = rates;  // old
config.playbackRates = rates; // new
```